docker-compose.template

```diff
@@ -36,17 +38,17 @@ hydroshare:
     SOLR_HOST: solr
   volumes:
     # hydroshare repository
-    - "HS_PATH:/home/docker/hydroshare"
+    - "HS_PATH:/hydroshare"
     # pycharm debugging
-    # - "/home/hydro/pycharm-debug:/home/docker/pycharm-debug"
+    # - "/home/hydro/pycharm-debug:/pycharm-debug"
     # log files
-    - "HS_LOG_FILES:/var/log/hydroshare"
+    - "HS_LOG_FILES:/hydroshare/log"
     # shared location for uwsgi.sock between containers
-    - "/var/run"
+    - "/hs_tmp"
     # temp directory shared with celery workers
-    - "/shared_temp"
+    - "/shared_tmp"
   ports:
-    - "1338:22"
+    - "1338:2022"
```

Comment:
@mjstealey Does the change of port mapping here prevent ssh connection to the container? What is the reason behind changing this port mapping here?